_snippets/typescript-sdk/create-thread.mdx

```diff
@@ -25,6 +25,14 @@ const res = await client.createThread({
   // Label types are created in settings where you can also copy
   // their ID.
   labelTypeIds: ['lt_01HB924PME9C0YWKW1N4AK3BZA'],
+  
+  // Thread fields are created in settings where you can also copy
+  // their key.
+  threadFields: [{
+    key: 'my_string_field',
+    type: ThreadFieldSchemaType.String,
+    stringValue: 'any value',
+  }],
```

Comment:
could use one example for boolean as well or even better for enum as well (which also uses stringValue)

Reply:
Added some examples below